Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -0,0 +1,152 @@
+within Buildings.Applications.DHC.CentralPlants.Cooling.Controls;
+model ChillerStage "Stage controller for chillers"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Modelica.SIunits.Time tWai "Waiting time";
+  parameter Modelica.SIunits.Power QEva_nominal
+    "Nominal cooling capaciaty (negative)";
+  parameter Modelica.SIunits.Power  criPoiLoa = 0.55*QEva_nominal
+    "Critical point of cooling load for switching one chiller on or off";
+  parameter Modelica.SIunits.Power  dQ = 0.25*QEva_nominal
+    "Deadband for critical point of cooling load";
+
+  Modelica.Blocks.Interfaces.RealInput QLoa(unit="W",max=0)
+    "Total cooling load, negative"
+    annotation (Placement(transformation(extent={{-140,-60},{-100,-20}})));
+  Modelica.Blocks.Interfaces.BooleanInput on "On signal of the chillers"
+    annotation (Placement(transformation(extent={{-140,20},{-100,60}})));
+  Modelica.Blocks.Interfaces.RealOutput y[2]
```

Comment:
better to change the output to a boolean output.
To find the boolean output, you can do as below:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/23491921/91774883-a8f1cb00-eb9e-11ea-81a3-279056bae5ee.png">
Basically, when the stage number is higher than zero, the chiller one should always be enabled. When the stage number is higher than one, the chiller two should be enabled.